store/src/java/com/zimbra/cs/dav/service/DavServlet.java

```diff
@@ -803,43 +806,48 @@ private boolean isProxyRequest(DavContext ctxt, DavMethod m) throws IOException,
         }
         ctxt.getResponse().setStatus(statusCode);
         ctxt.setStatus(statusCode);
-        try (InputStream in = httpResponse.getEntity().getContent()) {
-            switch (statusCode) {
-            case DavProtocol.STATUS_MULTI_STATUS:
-                // rewrite the <href> element in the response to point to local mountpoint.
-                try {
-                    Document response = W3cDomUtil.parseXMLToDom4jDocUsingSecureProcessing(in);
-                    Element top = response.getRootElement();
-                    for (Object responseObj : top.elements(DavElements.E_RESPONSE)) {
-                        if (!(responseObj instanceof Element)) {
-                            continue;
+        if (httpResponse.getEntity() != null && httpResponse.getEntity() != null)
```

Comment:
Two times check not required, you can remove one.

Reply:
my bad, made the corrections to have correct condition.